Makefile

```diff
@@ -596,12 +596,12 @@ third-party/catch/single_include/catch2/catch.hpp:
 
 # LCOV (hidden test target)
 .PHONY: lcov
-lcov: third-party/lcov/install/usr/local/bin/lcov
-third-party/lcov/install/usr/local/bin/lcov:
+lcov: third-party/lcov/install/bin/lcov
+third-party/lcov/install/bin/lcov:
 	@mkdir -p third-party
 	@cd third-party && \
 	git clone $(LCOV_URL) lcov --branch $(LCOV_VER) --depth=1
 	@cd third-party/lcov && \
 	mkdir -p install && \
-	CC=gcc CXX=g++ DESTDIR="install/" make install && \
+	CC=gcc CXX=g++ make PREFIX=`pwd`/install/ install && \
```

Comment:
I'd slightly prefer `$PWD` or even `$(pwd)`

Reply:
No objections to that. I just went with what I knew :)

Reply:
Oddly, $(PWD) and $(CURDIR) are both failing